### ISSUE TITLE
Authentication issue

### ISSUE THREAD

**Issue by vdhanasekaran** [neetcode-gh/leetcode#3167]
Hi Neetcode,

   I want to watch your course using my office laptop. My office laptop restricts social media authentication like Google, Facebook, LinkedIn, etc. Can you please implement username/password authentication also?

Reason for asking this: i will be using office laptop most of the time. i can spend some time in neetcode apart from doing office work if username/password authentication is present.  

Thanks,
Vivek

**Comment by Anybody2007** [comment]
Hi Devs, Please look into this as I am also going through this same issue.

**Comment by DarthMalak2003** [comment]
Yes Devs, I would also really appreciate it as it would help out a lot.

**Comment by papichuks** [comment]
this will be really helpful but not sure it has been implemented still.
